static/styles/zulip.css

```diff
@@ -1373,7 +1390,7 @@ td.pointer {
        private messages, mentions, and unread messages */
 
     .message-header-contents {
-        background-color: hsla(192, 19%, 75%, 0.2);
+        background-color: hsl(104.2, 100%, 92.5%);
```

Comment:
Temporary color -- the actual recipient header is being worked on in a separate PR.